modules/ROOT/pages/pricing-metrics.adoc

```diff
@@ -52,6 +52,10 @@ Anypoint Platform captures usage metrics across products, but not all metrics ar
 |Data Graph	
 |_DataGraph orchestration_: An API request made by Anypoint DataGraph to the source APIs to get data for the GraphQL API request made to Anypoint DataGraph. 
 |Orchestrations are not currently aggregated on usage reports.
+
+|IDP 
+|_Intelligent Document Processing (IDP) Document Pages_: A single page processed by IDP.
+|IDP document actions may process documents that have more than one page with each page counting separately. Executing document actions from RPA also counts Document Pages and additionally consumes the corresponding RPA Bot Minutes accounting for the time the RPA process runs. 
```

Comment:
```suggestion
|IDP document actions might process documents that have more than one page with each page counting separately. When RPA executes document actions, it also counts towards document pages and additionally consumes the corresponding RPA Bot Minutes accounting for the time the RPA process runs. 
```

Reply:
Thanks, Cris. Definitely easier to follow now. 
I still think you need commas before the clauses at the end of a couple of the sentences, because otherwise the sentence can be read incorrectly. Unfortunately, grammar checks aren't reliable for comma usage. For example, you need one between page and with in the first sentence and and between Minutes and accounting in the last sentence. I think you can safely follow this rule here: Use commas wherever necessary to prevent possible confusion or misreading.
If you don't have time today to address this today, I suggest asking the editorial team post-GA.